dds_web/scheduled_tasks.py

```diff
@@ -225,7 +225,7 @@ def monthly_usage():
     from dds_web import db
     from dds_web.database import models
     from dds_web.api.project import UserProjects
-    from dds_web.utils import page_query
+    from dds_web.utils import current_time, page_query
 
     with scheduler.app.app_context():
         # a mock dict with data that should be obtained from Safesprig's API
```

Comment:
Leave the `safespring_data` variable as it is, but you can remove lines 233-242 since the code is present in the for loops below now. 
`unit.safespring_name` on line 247 also needs to change since it's outside of a loop and `unit` is undefined (not caught until the above is removed)